schedule/schedule-appengine/src/main/client/jasify/admin/user/admin-user.html

```diff
@@ -33,6 +33,19 @@ <h2 translate="USERS_PROFILE" translate-values="{value: '{{::vm.user.name}}'}">
             <div role="tabpanel" class="tab-pane active" id="profile-tab">
                 <form role="form" novalidate name="vm.forms.userForm"
                       ng-submit="vm.forms.userForm.$valid && vm.submit()">
+                    <div jas-has-feedback class="form-group fg-float">
+                        <div class="fg-line">
+                            <input type="text" class="form-control fg-input" id="realName" name="realName"
+                                   ng-model="vm.user.realName">
+                        </div>
+                        <label class="fg-label">{{ 'FULL_NAME' | translate }}</label>
+                        <jas-feedback-icon-valid field="vm.forms.userForm.realName"></jas-feedback-icon-valid>
+                        <jas-feedback-icon-invalid field="vm.forms.userForm.realName"
+                                                   form="vm.forms.userForm"></jas-feedback-icon-invalid>
+                        <jas-error-help-block field="vm.forms.userForm.realName" form="vm.forms.userForm">
+                            <div ng-message="required"><span>{{ 'FULL_NAME_IS_REQUIRED' | translate }}</span></div>
```

Comment:
We should have a way of making a field required only if it wasn't empty before.  So, if field was unset and you don't touch it, you can still change other fields.

What i'm getting at here is, if you have a user who's name you don't know and you want to fix his e-mail (or something)  the user becomes locked and the admin will have to add some made-up name.